targets/TARGET_Realtek/TARGET_AMEBA/sleep_api.c

```diff
@@ -0,0 +1,36 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2013-2016 Realtek Semiconductor Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "sleep_api.h"
+#include "cmsis.h"
+
+// MCU SLEEP mode
+void sleep(void)
+{
```

Comment:
It's ok, we'd like to get that merged soon rather then have it all complete in one PR. To keep it clean please remove this file, and resubmit it, in a separate PR, when you wish to implement sleep functionality.